### PR TITLE
Added navigation UI layout

### PR DESCRIPTION
**Pull request by louisepujante**
[Notion Link](https://www.notion.so/Implement-navigation-UI-layout-11451f521fe64d009789361306b03fa5)

[Commands]
-`flutter pub get`

[Pre-conditions]
-none

[Expected Output]
-can view bottom navigation bar

[Notes]
-no behavior for pressing of buttons yet
-home icon has different color to display as if it's selected
-design implementation is not exactly the same. the bottom app bar with notch looks more rounded compared to the design. have tried researching on how to change rounded edges but did not see any solution for it :(

[Screenshots]
image

### CODE REVIEW

File: lib/views/widgets/navigation/navigation_button.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+// TODO: change widget to accept onPressed event
+class NavigationButton extends StatelessWidget {
+  final IconData icon;
+  final Color iconColor;
+
+  const NavigationButton(
+      {Key? key, required this.icon, this.iconColor = AppColor.lightGray})
+      : super(key: key);
```
Comment (inahsanjuan) [inline lib/views/widgets/navigation/navigation_button.dart]: Can I confirm is this the correct indentation here?

**Review by inahsanjuan — CHANGES_REQUESTED** [review]

File: lib/views/widgets/navigation/navigation_button.dart (hunk 1 above)
Comment (louisepujante) [reply lib/views/widgets/navigation/navigation_button.dart]: i think it be changed to multiline for easier readability miss, will change this..

**Review by inahsanjuan — APPROVED** [review]
* Can view all navigation icons
* Review code
image


Review time: 15 minutes
Comments: 1

**Review by ErickDoroy — APPROVED** [review]
## Approve #4

**Pull request merged**